src/harvest.cpp

```diff
@@ -82,7 +83,7 @@ const harvest_id &harvest_list::load( const JsonObject &jo, const std::string &s
     }
 
     if( jo.has_string( "message" ) ) {
-        ret.message_ = jo.get_string( "message" );
+        ret.message_ = to_translation( jo.get_string( "message" ) );
```

Comment:
```suggestion
        jo.read( "message", ret.message_ );
```
So other values such as context and translation comments can be read.